lesson07/assignment/fact-test.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link rel="stylesheet" href="css/style.css" />
+				<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/4.13.0/d3.min.js"></script>
+				<link rel="stylesheet" href="https://www.visualcrossing.com/widgets/forecast/weather-forecast-widget-d3.css">
+				<script src="https://cdn.jsdelivr.net/npm/vue@2/dist/vue.js"></script>
+        <script src="scripts.js"></script>
+        <title>Climate Fact</title>
+    </head>
+    <body>
+      <div class="container">
+        <header class="cell header">
+          <img src="./seattle_weather_logo.png" alt="Seattle Weather Logo in white color">
+        </header>
+        <nav class="nav" role="navigation" aria-label="primary">
+					<ul>
+						<li class="li"><a href="index.html">Home</a></li>
+						<li class="li"><a href="">Link</a></li>
+						<li class="li"><a href="weather-jokes.html">Weather Jokes</a></li>
+						<li class="li"><a href="">Climate Facts</a></li>
+					</ul>
+				</nav>
+        <!-- noaa fact -->
+          <main>
+						<div id="app">
+              <h2>Climate Facts</h2>
+              <p v-for="fact in factsList">{{ fact.title }}<br>
+								{{fact.descr}}</p>
+
+
+              <button v-on:click="randomize">Show me another fact</button>
+              <a v-bind:href="url">credit</a>
+						</div>
+
+          </main>
+          <!-- weather animiation module -->
+          <aside id=app class="cell aside" >aside
+          </aside>
+
+        <footer class="cell footer">Footer
+				</footer>
+      </div>
+
+      <script>
+        let facts = new Vue ({
+          el: '#app',
+          data: {
+            url: 'https://www.noaa.gov/climate',
+            factsList: [
+                { "title": "2019 was second-warmest year on record",
+								  "descr": "Earth’s global average surface temperature was 1.71°F (0.95°C) above the 20th-century average in 2019. Nine of the 10 warmest years on record have occurred since 2005."
```

Comment:
you should remove the extra space here and bring the `descr` to be lined up with `title`